Tools/autotest/common.py

```diff
@@ -1097,8 +1097,8 @@ def test_parameter_documentation_get_all_parameters(self):
             vehicle = "ArduCopter"
         if vehicle == "QuadPlane":
             vehicle = "ArduPlane"
-        cmd = [param_parse_filepath, '--vehicle', vehicle]
-#        cmd.append("--verbose")
+        cmd = [param_parse_filepath, '--vehicle', vehicle, "--format xml"]
```

Comment:
That should be
```suggestion
        cmd = [param_parse_filepath, '--vehicle', vehicle, "--format", "xml"]
```

... but I kind of like running the full generator here anyway.

Reply:
ok done !